lib/theme/view.js

```diff
@@ -1,11 +1,17 @@
 'use strict';
 
 const { dirname, extname, join } = require('path');
-const assignIn = require('lodash/assignIn');
-const omit = require('lodash/omit');
 const yfm = require('hexo-front-matter');
 const Promise = require('bluebird');
 
+const assignIn = (target, ...sources) => {
+  let source = [];
+  sources.forEach(src => {
+    source = source.concat([src, Object.getPrototypeOf(src)]);
```

Comment:
would `Array.push()` resolve the regression?

Reply:
Maybe. According to [this post](https://dev.to/uilicious/javascript-array-push-is-945x-faster-than-array-concat-1oki) `push()` is 945x faster than `concat()`.

I might try to re-implement `agginIn()` using `push()`.

Reply:
Looks like to https://github.com/hexojs/hexo/pull/3807/commits/f093749cde8e6973713547678662e15519c3c7e5 resolved the regression.